createdisk.sh

```diff
@@ -72,6 +72,9 @@ podman_version=$(${SSH} core@${VM_IP} -- 'rpm -q --qf %{version} podman')
 ${SSH} core@${VM_IP} -- 'sudo journalctl --rotate'
 ${SSH} core@${VM_IP} -- 'sudo journalctl --vacuum-time=1s'
 
+# Remove the base deployment from rpm-ostree
+${SSH} core@${VM_IP} -- 'sudo rpm-ostree cleanup -r'
```

Comment:
Do you know by how much this changes (reduces hopefully :) the size of the bundle?

Reply:
It reduced the size around 40MB with generated bundle.